### ISSUE TITLE
Vibration effect doesnt work on Android 13

### ISSUE THREAD

**Issue by Azim** [AnySoftKeyboard/AnySoftKeyboard#3471]
### Steps to reproduce
1. Enable vibration effect
2. Try typing
3. Observe phone not vibrating

### Expected behaviour
Vibration feedback on key press

### Actual behaviour
No vibration feedback on key press

**Android OS version:**
GrapheneOS build TP1A.220624.014.2022082400, Android 13

**Device manufacturer and model:**
Pixel 5a

**List of installed add-ons (like languages, or themes):**
Russian language pack

**Comment by NathanC** [comment]
Fair enough.  I can confirm that custom vibration does not work in 13 with system haptics disabled.

**Comment by Azim** [comment]
> It is working as intended "Use system vibration - Using Android system vibration setting".

Then why do i not get vibration when i disable system vibration and use "Use custom vibration setting" instead?
I expect custom vibration setting to vibrate even if android system's vibration is disabled
<details><summary>Screenshot</summary>
![image](https://user-images.githubusercontent.com/16478424/206224515-fb95eec8-fdc9-411d-b376-8133f9342972.png)
</details>

**Comment by menny** [comment]
that works for me on Android 13.
Which system-wide vibration settings have you disabled?

**Comment by Azim** [comment]
> Which system-wide vibration settings have you disabled?
"Touch Feedback"
<details><summary>screenshot</summary>

![image](https://user-images.githubusercontent.com/16478424/206252518-27d176dd-5683-470e-9b11-387610076c2d.png)

</details>

**Comment by menny** [comment]
So the intensity is set to zero. I am not sure I can do anything about that. I'll need to see how I can control the intensity (which means another slider in the settings)
